docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.6"
+
+services:
+  jupyter:
+    container_name: kglab-notebooks
+    build: .
+    ports:
+      - 8888:8888
+    volumes:
+      - ./examples:/home/jovyan/work/examples
+      - ./dat:/home/jovyan/work/dat
+      - ./kglab:/home/jovyan/work/kglab
```

Comment:
note that I don't _install_ kglab in the docker image. Instead, I just mount the notebooks, the data and the python files just like you would do if you were to run the examples locally : you first need to `git clone` the kglab repo to run the examples.